README.md

```diff
@@ -41,76 +34,77 @@ The current set of features include:
 - Clients and services
 - Timers
 - Composition (i.e. more than one node per process)
-- Time handling (system and steady, ROS time not yet supported https://github.com/ros2/ros2/issues/350)
+- Time handling (system and steady, ROS time not yet supported)
```

Comment:
Please revert this change, I like that there's a ticket attached to this item.

Reply:
I agree that it's nice to have a ticket attached, but I'll say that we should open a ticket against this repository, since most of the work to support time in rcljava will be here.  Then we can link this to the new ticket.

Reply:
I've created a ticket for ros2_java and added a reference here: 1028354